docker-for-mac/index.md

```diff
@@ -302,6 +302,9 @@ support is not included in Docker for Mac 17.12 CE Stable. To find out
 more about Stable and Edge channels and how to switch between them, see
 [General configuration](/docker-for-mac/#general).
 
+The addition of Kubernetes support increases the total binary size by about
```

Comment:
I misundestood your question on slack ; 300 MB is actually the size of the download when you activate Kubernetes ; the size of the D4Mac / Win install is increased by about 100 MB. (Not sure we need to mention this here).